src/api/product/services/getMany.js

```diff
@@ -2,7 +2,7 @@ import Product from '../../../models/product';
 import sendPaginatedList from '../../../utils/sendPaginatedList';
 import { sendErrorResponse } from '../../../utils/responseHelpers';
 
-async function readMany(req, res) {
+function readMany(req, res) {
```

Comment:
Why you removed async, if you remove async, and removed await from `await sendPaginatedList`, your catch will not catch error thrown by `sendPaginatedList`